<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDiskShow</class>
 <widget class="QDialog" name="UDiskShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qt200.qrc">
    <normaloff>:/resource/image/Title.png</normaloff>:/resource/image/Title.png</iconset>
  </property>
  <widget class="QListWidget" name="lwUDiskInfo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>261</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbUDiskTip">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRefresh">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbUDiskTag">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>U Disk</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cBoxUDiskTag">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="pBarUDiskHardPB">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qt200.qrc"/>
 </resources>
 <connections/>
</ui>
